charts/traces/README.md.gotmpl

```diff
@@ -0,0 +1,16 @@
+{{ template "chart.header" . }}
```

Comment:
@arthur-observe what is this template based on? any missing sections?

Reply:
Based on default template I removed the helm-docs version footer - you can see change here - charts/agent/README.md.gotmpl